### PR TITLE
Create a layout

### PR DESCRIPTION
**Pull request by mgmediaweb**
# Features

- Remove all elements from the installation of React JS.
- Create a layout of the Calculator with a grid system.
- Create a components button and screen (each with your style file).
- Import into the calculator component.
- All the Linters passed.

### CODE REVIEW

File: src/components/Calculator.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import Button from './button/Button';
+import Screen from './screen/Screen';
+import './Calculator.scss';
+
+function Calculator() {
+  const [screen, setScreen] = useState('0');
```
Comment (EricMbouwe) [inline src/components/Calculator.jsx]: - [x] For this project the Calculator.js component should be class-based. Let's fix that

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,13 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.scss';
 
 function App() {
   return (
```
Comment (EricMbouwe) [inline src/App.js]: - [x] The `App.js` component should be class-based. Let's fix that

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @mgmediaweb, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linters errors
- Good PR 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 2 above)
Comment (mgmediaweb) [reply src/App.js]: update the components required

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @mgmediaweb 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**